ampl-converter/src/main/java/com/powsybl/ampl/converter/version/ExtendedAmplExporterV2.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.ampl.converter.version;
+
+import com.powsybl.ampl.converter.AmplExportConfig;
+import com.powsybl.ampl.converter.AmplSubset;
+import com.powsybl.commons.io.table.Column;
+import com.powsybl.commons.io.table.TableFormatterHelper;
+import com.powsybl.commons.util.StringToIntMapper;
+import com.powsybl.iidm.network.Generator;
+import com.powsybl.iidm.network.LccConverterStation;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.util.HvdcUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.powsybl.ampl.converter.AmplConstants.*;
+
+/**
+ * 2nd extension of BasicAmplExporter, associated with AMPL version 1.2 (exporter id).
+ * The extension adds:
+ *  - A condenser boolean in the generator table.
+ *  - The load target Q in the lcc converter station table.
+ *  - TODO
```

Comment:
Don't forget to update this TODO comment.

Reply:
Done. Thank you for the comment.